index.html

```diff
@@ -2779,25 +2779,25 @@ <h4 id=role-map-none><code>none</code></h4>
     <tr>
       <th>MSAA + IAccessible2</th>
       <td>
-        <p>For objects that have required owned descendants (e.g., a grid owns gridcells, a list owns listitems), and the descendant is in the <a class="termref">accessibility tree</a>, expose it as <code>IA2_ROLE_TEXT_FRAME</code>. [=user agents=] SHOULD prune empty descendants from the <a class="termref">accessibility tree</a>.</p>
+        <p>For objects that have required accessibility children (e.g., a grid requires gridcell children, a list requires listitem children), and the descendant is in the <a class="termref">accessibility tree</a>, expose it as <code>IA2_ROLE_TEXT_FRAME</code>. [=user agents=] SHOULD prune empty descendants from the <a class="termref">accessibility tree</a>.</p>
```

Comment:
I think we should also remove the word "required" here (and in the following sections). IIRC part of the goal of this change was to remove the text around children being required in order to allow things like empty lists.

Reply:
ah whoops, nice catch thanks!